src/contracts/minter/facets/StabilityRateFacet.sol

```diff
@@ -88,22 +68,167 @@ contract StabilityRateFacet is DiamondModifiers {
             priceRateDelta: _setup.priceRateDelta,
             stabilityRate: uint128(WadRay.RAY)
         });
+
+        emit InterestRateEvent.StabilityRateConfigured(
+            _asset,
+            _setup.stabilityRateBase,
+            _setup.priceRateDelta,
+            _setup.rateSlope1,
+            _setup.rateSlope2
+        );
     }
 
     /**
      * @notice Configure existing stability rate values
      * @param _asset asset to configure
      * @param _setup setup parameters
      */
-    function configureStabilityRatesForAsset(address _asset, StabilityRateSetup memory _setup) external onlyOwner {
+    function updateStabilityRateParams(address _asset, StabilityRateParams memory _setup) external onlyOwner {
```

Comment:
We should add `ACTION.STABILITY_RATE` to the SafetyCouncil so we can manually turn on/off the stability rate. We need it for turning on/off the stability rate once liquidity in AMM has crossed certain levels and it's probably better to do it manually as opposed to programmatically given the different possible situations.

Reply:
Question: if the stability rate is 'off' because AMM liquidity isn't sufficient should we still maintain the base rate (in which case we can just update the existing params?) vs. should we have the ability to totally deactivate it?